src/backend/port/sysv_shmem.c

```diff
@@ -155,6 +155,8 @@ InternalIpcMemoryCreate(IpcMemoryKey memKey, Size size)
 		}
 	}
 #endif
+	if (am_wal_redo_postgres)
+		return valloc(size);
```

Comment:
Had to look up what `valloc` is, never heard of that before :-).

Why not malloc?

Reply:
Please add a comment